roles/kubernetes-apps/ansible/templates/coredns-config.yml.j2

```diff
@@ -22,17 +22,14 @@ data:
 {% endif %}
     .:53 {
         errors
-        health
+        health {
+            lameduck 5s
+        }
         ready
         kubernetes {{ dns_domain }} in-addr.arpa ip6.arpa {
           pods insecure
 {% if enable_coredns_k8s_endpoint_pod_names %}
           endpoint_pod_names
-{% endif %}
-{% if resolvconf_mode == 'host_resolvconf' and upstream_dns_servers is defined and upstream_dns_servers|length > 0 %}
-          upstream {{ upstream_dns_servers|join(' ') }}
-{% else %}
-          upstream /etc/resolv.conf
```

Comment:
why you remove upstream ?

Reply:
As said in the desc
> $ ./corefile-tool-amd64 deprecated --corefile corefile --from 1.3.0 --to 1.6.7
Option "upstream" in plugin "kubernetes" is deprecated in 1.4.x.
Option "upstream" in plugin "kubernetes" is ignored in 1.5.x.
[...]
Option "upstream" in plugin "kubernetes" is ignored in 1.6.x.
[...]
Option "lameduck" in plugin "health" is added as a default in 1.6.5.

`upstream` keyword is no longer used at all and ignored by coredns